src/connect/component.jsx

```diff
@@ -0,0 +1,42 @@
+/* @flow */
+import { loadAxo } from "@paypal/connect-loader-component";
+import {
+  getClientID,
+  getClientMetadataID,
+  getUserIDToken,
+} from "@paypal/sdk-client/src";
+
+// TODO: What's the expected structure/approach for this interface. It's not a zoid
+// scenario, so what do we return?
+// -> Looks like it returns a function that accepts the props
+// How do we define the input of merchant params here?
+export const getConnectComponent = async (merchantProps) => {
+  const cmid = getClientMetadataID();
+  const clientID = getClientID();
+  const userIdToken = getUserIDToken();
+  // TODO: Sort out integration specifics for inputs
+  try {
+    const loadResult = await loadAxo({
+      client: { getVersion: () => "3.97.3-connect-alpha.6.1" }, // this is currently supported for BT
+      sdkVersion: "", // PPCP proposal: string instead of function
+      minified: false,
```

Comment:
```suggestion
      platform: "PPCP",
      sdkVersion: "3.97.3-connect-alpha.6.1",
      minified: false,
```

Reply:
@siddy2181 I updated the `sdkVersion` to -> `btSdkVersion` to be more specific since that's what we're attempting to load under the hood